db/models.py

```diff
@@ -0,0 +1,99 @@
+"""Models for Mongo DB"""
+
+from datetime import date, datetime
+from pathlib import Path
+from typing import Dict, List, Optional
+from pydantic import BaseModel, Field
+
+
+class DocumentBaseModel(BaseModel):
+    """Base model for Mongo documents"""
+
+    id: str = Field(None, alias="_id")
+    inserted_at: Optional[datetime]
+    updated_at: Optional[datetime]
+
+    def dict(self, **kwargs) -> Dict:
+        """Return a dictionary representation of the model"""
+        values = super().dict(**kwargs)
+        if "id" in values and values["id"]:
+            values["_id"] = values["id"]
+        if "exclude" in kwargs and "_id" in kwargs["exclude"]:
+            del values["_id"]
+        return values
+
+
+class CircuitScheduleDoc(BaseModel):
+    """EVC circuit schedule model"""
+
+    id: str
+    date: Optional[date]
+    frequency: Optional[str]
+    interval: Optional[int]
+    action: str
+
+
+class TAGDoc(BaseModel):
+    tag_type: int
+    value: int
+
+
+class UNIDoc(BaseModel):
+    """UNI model"""
+
+    tag: Optional[TAGDoc]
+    interface_id: str
+
+        
+class EVCBaseDoc(DocumentBaseModel):
+    """Base model for EVC documents"""
+
+    uni_a: UNIDoc
+    uni_z: UNIDoc
+    name: str
+    start_date: Optional[datetime]
```

Comment:
I realized that certain `datetime` values such as `start_date`, `end_date` `creation_time` when they're being serialized as JSON with the default serializer that Flask uses the format now would result like as "Tue, 05 Jul 2022 21:55:52 GMT", and the string content used to isoformat compatible like `2022-07-05T17:17:40`, so this could break the API content depending how clients would be parsing this.  @ajoaoff maybe the simplest way to fix, stardandize would be to open an issue and overwrite the default serailizer from Flask in the `app` instance on core and implement the expected serialization for datetime values as iso string? Let me know what you think.

Reply:
I've mapped it here https://github.com/kytos-ng/kytos/issues/244